docker-compose.yml

```diff
@@ -14,7 +14,6 @@
 # Run migrations: docker-compose run migrate{{/db.is_sqlite}}{{/fluent}}
 #       Stop all: docker-compose down{{#fluent}}{{^db.is_sqlite}} (add -v to wipe db){{/db.is_sqlite}}{{/fluent}}
 #
-version: '3.7'{{#fluent}}{{^db.is_sqlite}}
```

Comment:
We still need the `{{#fluent}}{{^db.is_sqlite}}` section other this breaks the template